app/views/kudos/index.html.erb

```diff
@@ -11,7 +11,7 @@
             <h6 class="card-subtitle mb-2 text-muted">Value: <%= kudo.company_value.title %></h6>
             <td><%= link_to 'Show', kudo, class:"btn btn-sm btn-outline-secondary" %></td>
         
-            <% if kudo.employee == current_employee %>
+            <% if policy(kudo).update? %>
```

Comment:
I think without it you can see the edit buttons for other users' kudos
```suggestion
            <% if policy(kudo).update? && kudo.employee == current_employee %>
```

Reply:
Fixed.